docs/decisions/0026-file-service.md

```diff
@@ -0,0 +1,144 @@
+---
+# These are optional elements. Feel free to remove any of them.
+status: proposed
+contact: crickman, mabolan, semenshi
+date: 2024-01-16
+---
+
+# File Services
+
+## Context and Problem Statement
+OpenAI provides a file service for uploading files to be used for *assistant retrieval* or *model fine-tuning*: `https://api.openai.com/v1/files`
```

Comment:
Probably better to link to the doc on Files rather than the API endpoint itself:

https://platform.openai.com/docs/api-reference/files

Reply:
I avoided since the API doc URL's have exhibited a pattern of shifting around over time and create link-check problems.

The provided endpoint _is_ the service, after all.

Reply:
OK. I'll leave it to your discretion. But just bear in my that putting the API URL is not very useful because when anyone clicks on it, they'll just get a weird "page" not actually meant for direct human consumption giving them an error.